docs/source/guide/explanation/additional_features/hpo.rst

```diff
@@ -1,4 +1,184 @@
 Hyperparameters Optimization
 ============================
 
-Coming soon...
\ No newline at end of file
+Hyper-parameter optimization (HPO) can be a time-consuming process, even with state-of-the-art off-the-shelf libraries. OpenVINO™ Training Extensions makes HPO faster and easier by providing an easy-to-use interface and automatic configuration.
+
+With OpenVINO™ Training Extensions, you can run hyper-parameter optimization by simply adding a time constraint parameter. The auto-config feature automatically sets internal control parameters, guaranteeing that HPO will finish within the given time constraint.
+
+OpenVINO™ Training Extensions provides both sequential and parallel methods, making it scalable for different training environments. If you have multiple GPUs, you can accelerate HPO by utilizing all available GPU resources.
+
+Key features of OpenVINO™ Training Extensions include:
+
+- **Easy usability** : By using time as the control parameter, OpenVINO™ Training Extensions offers a straightforward and intuitive interface for users.
+
+- **Auto-config** : The automatic configuration feature sets internal control parameters automatically, ensuring that HPO finishes within the given time constraint.
+
+- **Scalability** : OpenVINO™ Training Extensions offers both sequential and parallel methods, making it scalable for different training environments. If you have multiple GPUs, you can take advantage of all available GPU resources to accelerate HPO.
+
+**************
+Simpe HPO Exapmle
```

Comment:
It should be located in tutorials/advanced/
The explanation section is only for describing algorithm

1) Please, add description about what algorithm is used for HPO with link to articles and briefly how it works
2) Add overview of the HPO config and what parameters can be optimized by HPO and how to set them (how to set time limits, lower and higher boundaries for parameters, etc.)
3) Add just one line general example how to enable it here

Everything else should be moved to tutorials

Reply:
I applied your comment. Could you check it?